docs/source/examples/relaxation.py

```diff
@@ -0,0 +1,38 @@
+"Relaxation examples"
+
+from gpkit import Variable, Model
+x = Variable("x")
+x_min = Variable("x_min", 2)
+x_max = Variable("x_max", 1)
+m = Model(x, [x <= x_max, x >= x_min])
+print "Original model"
+print "=============="
+print m
+print
+# m.solve()  # raises a RuntimeWarning!
+
+print "With constraints relaxed equally"
+print "================================"
+from gpkit.constraints.relax import RelaxAll
+relaxall = RelaxAll(m)
```

Comment:
there was a miscommunication above; I was confused by the names. My point was that usually it doesn't make sense to relax all the constraints.

I think the current naming does not make clear the available distinctions, namely:

 - one might want to relax only a subset of the constraints rather than all the constraints
 - one might want the relaxation variables to be shared or independent (currently named `RelaxAll` and `RelaxConstraints`, respectively.